src/plugins/intel_npu/src/plugin/npuw/just_sync_infer_request.hpp

```diff
@@ -148,8 +149,19 @@ class JustInferRequest final : public IBaseInferRequest {
     };
     std::vector<GlobalIO> m_subrequests_gio;
 
+    // BTW, this mutex makes no sense - it is not shared among other concurrent requests.
+    // it should've been static for this purpose.
     std::mutex m_alloc_mutex;
```

Comment:
Good catch! Could you update it? I believe we suffer the same issue in weights bank :(

Reply:
What should I update here?

Reply:
Oh, nvm. For the bank we only have 1 instance, so that's not an issue. Here we could use `inline static mutex` but I suggest we just drop it since there is no concurrent allocation and this mutex does nothing if it'll be required one day, it's simply misleading

Reply:
Dropped